nautobot/core/filters.py

```diff
@@ -629,15 +629,23 @@ def _generate_lookup_expression_filters(cls, filter_name, filter_field):
         if field is None:
             return magic_filters
 
+        # If the field allows null values, add an `isnull`` check
+        if field.null and field.get_internal_type() in constants.FILTER_LOCAL_FIELDS:
+            # Use this method vs extend as the `lookup_map` variable is generally one of
+            # the constants which we do not want to update
+            lookup_map = dict(lookup_map, **{"isnull": "isnull"})
```

Comment:
Similarly here since the code is trying to discern and set the lookup value also be the string `"isnull"`, it feels like it would be more readable if we didn't just reference "local fields".

Reply:
premature optimization, should be good now.